OpenRA.Game/Orders/UnitOrderGenerator.cs

```diff
@@ -85,7 +86,7 @@ public virtual string GetCursor(World world, CPos cell, int2 worldPixel, MouseIn
 				    (mi.Modifiers.HasModifier(Modifiers.Shift) || !world.Selection.Actors.Any());
 			}
 
-			return useSelect ? "select" : "default";
+			return useSelect ? ChromeMetrics.Get<string>("WorldSelectCursor") : ChromeMetrics.Get<string>("WorldDefaultCursor");
```

Comment:
`ChromeMetrics.Get` does a lot of work (it uses FieldLoader to parse the value every time you call it!), so we really want to avoid calling this every time.

You can define
```csharp
readonly string worldSelectCursor = ChromeMetrics.Get<string>("WorldSelectCursor");
readonly string worldDefaultCursor = ChromeMetrics.Get<string>("WorldDefaultCursor");
```

at the top of the class and return the cached values here, and likewise in `Widget`, `PlaceBuildingOrderGenerator`, `RadarWidget`.

Reply:
I would avoid the temptation to define a generic public `Cursor` on the base widget, i suspect this will cause more trouble than it is worth.

Reply:
Oh, yes that would be bad for performance. Fixed.